### ISSUE TITLE
Does the script work in the terminal (command-line usage)?

### ISSUE THREAD

**Issue by ghost** [donmaiq/Mpv-Playlistmanager#98]
Shift + Enter progresses to the next track / playlist entry when running mpv on the command-line, no mpv-playlistmanager's interface appears. The script functions correctly in the mpv window (which I prefer not to launch when playing audio).

I exclusively play audio with the mpv's `--no-video` flag (disables video playback, the mpv windows isn't created) and more or less always launch mpv (add files to a playlist) via the Unix shell.

**Comment by jonniek** [comment]
It currently does not work because terminal will not render ASS tags. It would not be impossible to fallback to OSD messages when playing audio only. If you look at commits before the ASS tags were implemented they should work on terminal. Here is the latest playlistmanager with OSD rendering:

https://github.com/jonniek/mpv-playlistmanager/tree/1e12d1c79954afc3fea8dd46c0bd24a6a4300164

https://raw.githubusercontent.com/jonniek/mpv-playlistmanager/1e12d1c79954afc3fea8dd46c0bd24a6a4300164/playlistmanager.lua

Above version renders like this in my terminal:
![image](https://user-images.githubusercontent.com/16519907/205551062-a2578a20-0f77-4937-8b8c-293b4962aa6a.png)

It's missing a lot of features of course, but maybe it can work temporarily.
I had to change the SHIFT+ENTER keybind though, as that was not working for some reason.

I'll leave this issue open since it would be a good feature to fallback to OSD rendering when no video is playing. Shouldn't be an enormous effort.
